frontend/components/AddressForm/index.jsx

```diff
@@ -106,6 +107,15 @@ export class AddressForm extends Component {
       EventEmitter.emit(NAVIGATOR_SAVE_BUTTON_SHOW);
       EventEmitter.emit(NAVIGATOR_SAVE_BUTTON_DISABLE);
     }
+
+    // Init default tags for first address
+    if (this.props.isFirstAddress) {
+      const tags = [];
+      this.props.config.splitDefaultAddressesByTags.forEach(tag => (
```

Comment:
`splitDefaultAddressesByTags` should be renamed as suggested in the comment within the `shopgate.user.getConfig.v1` pipeline definition